ppb/__init__.py

```diff
@@ -58,20 +59,18 @@
 )
 
 
-def _make_kwargs(setup, title, engine_opts):
+def _make_kwargs(starting_scene, title, engine_opts):
     kwargs = {
         "resolution": (800, 600),
-        "scene_kwargs": {
-            "set_up": setup,
-        },
+        "first_scene": starting_scene,
         "window_title": title,
         **engine_opts
     }
     return kwargs
 
 
-def run(setup: Callable[[BaseScene], None] = None, *, log_level=logging.WARNING,
-        starting_scene=BaseScene, title="PursuedPyBear", **engine_opts):
+def run(setup: Union[Callable[[BaseScene], None], BaseScene] = lambda scene: None, *, log_level=logging.WARNING,
+        starting_scene=None, scene_class=None, title="PursuedPyBear", **engine_opts):
```

Comment:
This seems like a silly deprecation?

I think the rename of `starting_scene` -> `scene_class` reduces clarity (slightly) and doesn't represent any meaningful improvement?

Reply:
Fair. I think my issue was the name shadowing, but reasonable.